app.js

```diff
@@ -3,9 +3,22 @@ import { wizardScreen } from './js/screens/wizard-screen.js';
 
 document.addEventListener('DOMContentLoaded', () => {
    const app = document.querySelector('#app');
+   //margins
+   const topMargin = document.createElement('img');
```

Comment:
**nitpick(non-blocking):** Took me a while to understand why you didn't use CSS for setting margins :D But now I see it's that image bar on top, and the bottom of the app :D I think you can set more descriptive name here